package.json

```diff
@@ -8,9 +8,12 @@
   "scripts": {
     "bump-gradle-version": "node ./scripts/gradle-version-update.js --package-version=${npm_package_version} && git add app/build.gradle",
     "build": "./gradlew clean assembleDebug && npm run move-apk",
+	"build:win": "gradlew clean assembleDebug && npm run create-folder && npm run copy-apk",
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
     "move-apk": "mkdir -p apks && cp app/build/outputs/apk/debug/app-debug.apk ./apks/ApiDemos-debug.apk",
+	"create-folder": "if not exist apks (mkdir apks) else (echo Folder Already Exist!)",
```

Comment:
I would rather move these commands into a separate `.js` script, so the task for building on Windows would look like `node ./scripts/build-win.js` and then this script itself should contain the necessary primitives

Reply:
I move the "create-folder" command to seperate .js as requested.
Do u want me to move the copy-apk as well?

Reply:
yes, I would prefer to move the whole building logic into this script.

I am still not sure why it is necessary to add all these changes though. I would simply use cygwin or WSL there to build the project

Reply:
just a small convenient approach. I took it more as a chance to learn some node.js :)